src/content/documentation/publish/add-on-policies-preview-2025-08.md

```diff
@@ -1,303 +0,0 @@
----
```

Comment:
I think we need to keep this URL since it has been shared previously. What if we start by keeping the content, and we just say something like:

> The policies are available at: <link>